test/erlcloud_cloudformation_tests.erl

```diff
@@ -0,0 +1,544 @@
+          <member>
+            <StackName>My Stack</StackName>
+            <StackId>Stack Id</StackId>
+            <CreationTime>Not Today</CreationTime>
+            <StackStatus>status</StackStatus>
+            <DisableRollback>false</DisableRollback>
+            <Outputs>
+              <member>
+                <OutputKey>Some Key</OutputKey>
+                <OutputValue>Output</OutputValue>
+              </member>
+            </Outputs>
+          </member>
+        </Stacks>
+      </DescribeStacksResult>
+      <ResponseMetadata>
+        <RequestId>request ID</RequestId>
+      </ResponseMetadata>
+    </DescribeStacksResponse>">>,
+
+    Expected = [[{stacks, [[
+            {member, [[{stack_name, "My Stack"},
+                       {stack_id, "Stack Id"},
+                       {creation_time, "Not Today"},
+                       {stack_status, "status"},
+                       {disable_rollback, "false"},
+                       {outputs, [[
+                            {member, [[
+                                {output_key, "Some Key"},
+                                {output_value, "Output"}
+                            ]]}
+                        ]]}
+            ]]}
+        ]]},{response_meta, [[{request_id, "request ID"}]]}]],
+
+    output_test(?_f(erlcloud_cloudformation:describe_stacks_all(
+                [{"StackName", "Some Stack Name"}], Config = #aws_config{}
+            )),
+        {?LINE, {"Test describe all stacks", Response, Expected}}).
+
+describe_stack_resources_output_tests(_) ->
+    Response = <<"<DescribeStackResourcesResponse xmlns=\"http://cloudformation.amazonaws.com/doc/2010-05-15/\">
+      <DescribeStackResourcesResult>
+        <StackResources>
+          <member>
+            <StackId>Stack ID</StackId>
+            <StackName>My Stack</StackName>
+            <LogicalResourceId>MyDBInstance</LogicalResourceId>
+            <PhysicalResourceId>Resource ID</PhysicalResourceId>
+            <ResourceType>Resource Type</ResourceType>
+            <Timestamp>Timestamp</Timestamp>
+            <ResourceStatus>Status</ResourceStatus>
+          </member>
+        </StackResources>
+      </DescribeStackResourcesResult>
+      <ResponseMetadata>
+        <RequestId>Request ID</RequestId>
+      </ResponseMetadata>
+    </DescribeStackResourcesResponse>">>,
+
+    Expected = [[{resources, [[{member, [[
+        {stack_id, "Stack ID"},
+        {stack_name, "My Stack"},
+        {logical_resource_id, "MyDBInstance"},
+        {physical_resource_id, "Resource ID"},
+        {resource_type, "Resource Type"},
+        {timestamp, "Timestamp"},
+        {resource_status, "Status"}]]}]]},
+    {response_meta, [[{request_id, "Request ID"}]]}]
+    ],
+
+    output_test(?_f(erlcloud_cloudformation:describe_stack_resources(
+                [{"StackName", "Some Stack Name"}], Config = #aws_config{}
+            )),
+        {?LINE, {"Test describe stack resources", Response, Expected}}).
```

Comment:
so basically I want these tests to have exactly  the same format as in rds_tests: using `?cf_tests`, `?f` and other things